src/test/ui/generic-associated-types/issue-68656-unsized-values.stderr

```diff
@@ -8,15 +8,15 @@ LL | #![feature(generic_associated_types)]
    = note: see issue #44265 <https://github.com/rust-lang/rust/issues/44265> for more information
 
 error[E0271]: type mismatch resolving `<T as Deref>::Target == T`
-  --> $DIR/issue-68656-unsized-values.rs:16:5
+  --> $DIR/issue-68656-unsized-values.rs:16:21
    |
 LL |     type Item<'a>: std::ops::Deref<Target = T>;
    |                                    ---------- required by this bound in `UnsafeCopy::Item`
 ...
 LL | impl<T: Copy + std::ops::Deref> UnsafeCopy<T> for T {
    |      - this type parameter
 LL |     type Item<'a> = T;
-   |     ^^^^^^^^^^^^^^^^^^ expected type parameter `T`, found associated type
+   |                     ^ expected type parameter `T`, found associated type
```

Comment:
This seems somewhat confusing, but thankfully we provide a structured suggestion.